### ISSUE TITLE
Simulation not running at all

### ISSUE THREAD

**Issue by dgebel** [the3dadvantage/Modeling-Cloth-2_8#13]
Running Release version of 2.8 and the version of Modeling Cloth as of last evening, on Windows 10 home.

First though, thanks so much for making this donation-ware Rich. The video for this looks fantastic - but I'm having a total show-stopper.

I tried to follow the steps to get it going, followed through the action steps in the video (linked from Blendernation) several times (not that there's many steps thankfully!) but no matter what I do, I can't get the simulation to actually run. 

I can't grab the cloth like you do in the video, the plane stays perfectly flat, the cloth doesn't wrinkle up, adding wind, etc. doesn't do anything, nothing happens with the Grab button either. I definitely have Continuous Update switched on.  The shape keys are created. I tried subdividing manually & with modifier, applying and not applying it...

Pins are the only thing that seems to function - I can add, move & delete pins. Of course, with no simulation, I can't see what effect they should have.

Attaching (I hope) the basic file I created in hopes that it will help show someone what my issue is!

[ModCloth1.zip](https://github.com/the3dadvantage/Modeling-Cloth-2_8/files/3601639/ModCloth1.zip)

**Comment by the3dadvantage** [comment]
Hello, dgebel. How do you pronounce that by the way?

Just letting you know I opened the blend file and I can't get it to not
work. I also don't get any error messages.
It could be a windows issue but it's probably something more simple.
If you can link me any error messages you see that would be helpful. Maybe
just pull up the system console window and take a screenshot.
There are also some build conflicts I've run into. Some versions of blender
2.8 don't work.
Let's see if we can figure it out.

Best!

**Comment by dgebel** [comment]
Thanks for the quick response!  (Actually, the name is simpler than that - D. G. Ebel so no complicated pronunciation. Although my doctor is Ndubsi so I can see how you might have been confused)

Here's the error I get when I try to use the grab button on the mesh. It appears when I actually move the mouse after clicking on the mesh.

![image](https://user-images.githubusercontent.com/36416022/64794727-efdc5000-d54a-11e9-9349-afbfe529bfa4.png)

Windows hides the system console - I can't find it after the program loads. It must be around somewhere, because it reappears very briefly when existing the program.

**Comment by the3dadvantage** [comment]
It might be something to do with deleting an object that has collision or
modeling cloth enabled. I'm reworking how I handle deleted objects.
Try hitting f8 to reload the addon. see if that helps.
You can usually get the system console to pop up if you open the search
menu and type "togggle system console." Sometimes it pops up behind other
windows so you have to check for that...
If you still can't get it working we can set up a screen share through
skype. Maybe I'll notice something.
my skype handle is: thanetasha

Cheers!

**Comment by dgebel** [comment]
Finally got it to work!
I cleared out a bunch of scripts causing errors I found in the system console. I think one of them must have been interfering. Either because it was an error or a fundamental conflict. BlendKit was one of them, and then there were a couple that said they didn't exist, so maybe they were left over from a prior 2.8 beta installation and I didn't reinstall them with the production version.

However, I had to create the simulation from scratch again. When I tried the sample file I sent you, it still had a the same problem.  Here's the system console dump:
---------------------
Read prefs: C:\Users\Grant\AppData\Roaming\Blender Foundation\Blender\2.80\config\userpref.blend
found bundled python: C:\Program Files\Blender Foundation\Blender\2.80\python
Imported Modeling Cloth
Read blend: D:\Zed\assets\blender\Blender Projects\ModCloth1.blend
Traceback (most recent call last):
  File "C:\Users\Grant\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\Modeling-Cloth-2_8-master\ModelingCloth28.py", line 2604, in modal
    main_drag(context, event)
  File "C:\Users\Grant\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons\Modeling-Cloth-2_8-master\ModelingCloth28.py", line 2564, in main_drag
    data[best_obj.name].clicked = True
KeyError: 'Plane'

location: <unknown location>:-1
-----------------------

Still have a fair number of problems with the simulation stopping with some errors after awhile though. Will have to sort through them later though. Probably create a different issue for them.

Thanks!
